File: tests/human/x/zig/break_continue.zig
const std = @import("std");

pub fn main() void {
    var numbers = [_]i32{1,2,3,4,5,6,7,8,9};
    for (numbers) |n| {
        if (n % 2 == 0) continue;
        if (n > 7) break;
        std.debug.print("odd number: {d}\n", .{n});
    }
}

File: tests/human/x/cpp/break_continue.cpp
#include <iostream>
#include <vector>
int main(){
    std::vector<int> numbers{1,2,3,4,5,6,7,8,9};
    for(int n: numbers){
        if(n % 2 == 0) continue;
        if(n > 7) break;
        std::cout << "odd number: " << n << std::endl;
    }
    return 0;
}
